pytorch_lightning/loggers/wandb.py

```diff
@@ -490,6 +505,7 @@ def _scan_and_log_checkpoints(self, checkpoint_callback: "ReferenceType[ModelChe
                             "save_top_k",
                             "save_weights_only",
                             "_every_n_train_steps",
+                            "_every_n_val_epochs",
                         ]
                         # ensure it does not break if `ModelCheckpoint` args change
                         if hasattr(checkpoint_callback, k)
```

Comment:
quick ques: any reason why we need to have this check here?
if something is changed in ModelCheckpoint, shouldn't the change be made here considering both of them are on master.

Reply:
It's just an extra security, there are so many cases to consider that I  just wanted to make sure it would not break